apps/ledger-live-mobile/src/components/TranslatedError/hooks/useErrorLinks.tsx

```diff
@@ -0,0 +1,52 @@
+import { useNavigation } from "@react-navigation/core";
+import React, { useCallback, useMemo } from "react";
+import { Linking, Text } from "react-native";
+import { ScreenName } from "../../../const";
+
+export function useErrorLinks(error?: Error | null) {
+  const { navigate } = useNavigation();
+
+  const onErrorLinkPress = useCallback(
+    async (errorLink: string) => {
+      if (errorLink.startsWith("http")) {
+        const canOpenURL = await Linking.canOpenURL(errorLink);
+
+        if (canOpenURL) {
+          Linking.openURL(errorLink);
+          return;
+        }
+      }
+
+      if (errorLink === "/platform/multibuy") {
+        navigate(ScreenName.PlatformApp, {
+          platform: "multibuy",
+        });
```

Comment:
having specific routing logic inside a generic hook feels wrong,
is there a way to put the routing logic back where this error link is used? 🤔  alternatively is there a way to centralize the routing outside of the hook? does `navigate("/platform/multibuy")`  actually work? i notice we sometimes use navigate(string) on the deeplinking logic

Reply:
Will try with just the string, agreed it feels a bit wrong!